manifests/install.pp

```diff
@@ -75,13 +75,20 @@
     $_package_options = {}
   }
 
+  if $::osfamily =~ /windows/ {
```

Comment:
Meaning it would take more work, and I don't have time to tackle that right now.